app/components/viral/form/prefixed/boolean_component.html.erb

```diff
@@ -0,0 +1,62 @@
+<div class="flex">
+  <span
+    class="
+      inline-flex
+      items-center
+      px-3
+      text-sm
+      text-slate-900
+      bg-slate-200
+      border
+      rounded-e-0
+      border-slate-300
+      rounded-s-md
+      dark:bg-slate-600
+      dark:text-slate-400
+      dark:border-slate-600
+      font-mono
+    "
+  >
+    <%= prefix %>
+  </span>
+  <div
+    class="
+      rounded-none
+      rounded-e-lg
+      bg-slate-50
+      border
+      text-slate-900
+      block
+      flex-1
+      min-w-0
+      w-full
+      text-sm
+      border-slate-300
+      p-2.5
+      dark:bg-slate-700
+      dark:border-slate-600
+      dark:placeholder-slate-400
+      dark:text-white
+      dark:focus:ring-primary-500
+      dark:focus:border-primary-500
+    "
+  ><div class="space-x-4">
+      <span>
```

Comment:
**nitpick (blocking):** Could you update the radio button styling to match the flowbite styling found [here](https://flowbite.com/docs/forms/radio/#radio-example).

Reply:
Fixed in e5451af78b230b6426e0052cbd59bb285b8185a0